lisk/tslint.json

```diff
@@ -1,6 +1,8 @@
 {
 	"extends": ["../tslint.json"],
 	"rules": {
-		"no-null-keyword": false
+		"no-null-keyword": false,
+		"no-let": false,
+		"no-console": false
```

Comment:
@lsilvs We don't have a proper logger at the moment, thus still using console statements. Until we introduce a logger, I decided to disable that rule instead of suppressing the error messages.